tox.ini

```
@@ -23,6 +24,7 @@ deps =
     django22: Django>=2.2.1,<2.3
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django31: Django>=3.2rc1,<3.1
```

Will update this once django 3.2 releases.

```suggestion
    django31: Django>=3.2,<3.1
```